src/Functions/determineElectrodeType.m

```diff
@@ -1,12 +1,25 @@
-%% determineElectrodeType - get most likly electrode type be eucleadian distance between detected ans specified peaks
+function [elecStruct, rms, d] = determineElectrodeType(peakDistances)
+% gets the most suitable electrode type based on the euclidean distance between 
+% detected and specified peaks in the collected data. Data should should be provided 
+% in the form of the file 'electrodeGeometries.mat'. 
+%
+% USAGE:
+%
+%    [electStruct, rms, d] = determineElectrodeType(peakDistances)
 %
-% Andreas Husch
-% Centre Hospitalier de Luxembourg / Luxembourg Centre for Systems
-% Biomedicine, University of Luxembourg
-% 2016  - 2017
-% mail@andreashusch.de
+% INPUT: 
+%    peakDistances:     The peak distances between the signals, provided by the inserted 
+%                       data sheet.   
+%
+% OUTPUTS: 
+%    electStruct:       Electrode Type.
+%    rms:               Root-mean-square of the distance between the electrode signals.
+%    d:                 ?
```

Comment:
@adhusch, we need your input here...

Reply:
I will look into this, i didn't notice i left the question mark in it until after we had our meeting

Reply:
d: euclidian distance to the closest electrode prototype